src/azul/__init__.py

```diff
@@ -1322,6 +1322,15 @@ def monitoring_topic_name(self):
     def cloudwatch_dashboard_template(self) -> str:
         return f'{config.project_root}/terraform/cloudwatch_dashboard.template.json'
 
+    @property
+    def account_alt_contact_security(self) -> dict[str]:
+        value = self.environ.get('azul_account_alt_contact_security')
+        if value is None:
+            return dict()
```

Comment:
Any false-ish value would do since you don't actually interpolate anything when the value evaluates to False. The most obvious false-ish value is None.